### PR TITLE
Changes has been made for donate feature

### PR DESCRIPTION
**Pull request by bhushano7**

### CODE REVIEW

File: public/js/pages/common.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-var domainUrl = 'http://localhost:1234';
+var domainUrl = 'http://' + window.location.host;
 var commonData = {
     "Preference": {
         "InfoEmail": "bhushan2250@gmail.com"
     }
-};
\ No newline at end of file
+}
+var stripePublisherKey = "pk_test_ULZHqL27xD6Zd0j3gnro1eY6";
```
Comment (AlexeyAnshakov) [inline public/js/pages/common.js]: Are you sure it's secure?

File: public/js/pages/common.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 //var domainUrl = window.location.origin
 //domainUrl = domainUrl.replace("8081", "1234");
-var domainUrl = 'http://localhost:1234';
+var domainUrl = 'http://' + window.location.host;
 var commonData = {
     "Preference": {
         "InfoEmail": "bhushan2250@gmail.com"
```
Comment (AlexeyAnshakov) [inline public/js/pages/common.js]: change on info@webrunes.com

File: widget/iframe.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (AlexeyAnshakov) [inline widget/iframe.html]: Why do we need this page?

File: public/js/pages/common.js (hunk 1 above)
Comment (bhushano7) [inline public/js/pages/common.js]: This public use for only generate token for process credit card, the credit card process can be completed by using only secrete key. This secret key is inside our config.js which is secured.

File: public/js/pages/common.js (hunk 2 above)
Comment (bhushano7) [inline public/js/pages/common.js]: I will update this in next merge.

File: widget/iframe.html (hunk 3 above)
Comment (bhushano7) [inline widget/iframe.html]: This is doc type, which is used to identify this is page written using html5.

**Comment by AlexeyAnshakov** [comment]
@bhushano7 , please write proper name (more informative) and resolve conflicts

**Pull request merged**